src/Models/BaseElement.php

```diff
@@ -1066,28 +1061,13 @@ public static function getBlockConfig()
         return [];
     }
 
-    /**
-     * The block actions is an associative array available for providing data to the client side to be used to describe
-     * actions that may be performed. This is available as a plain "ObjectType" in the GraphQL schema.
-     *
-     * By default the only action is "edit" which is simply the URL where the block may be edited.
-     *
-     * To modify the actions, either use the extension point or overload the `provideBlockSchema` method.
-     *
-     * @internal This API may change in future. Treat this as a `final` method.
-     * @return array
-     */
-    public function getBlockSchema()
+    public function getBlockSchema(): array
     {
-        $blockSchema = $this->provideBlockSchema();
-
-        $this->extend('updateBlockSchema', $blockSchema);
-
-        return $blockSchema;
+        return $this->provideBlockSchema();
```

Comment:
There's no sense having a method that just returns the value of another method - get rid of one or the other (or put the extension hook back)

Reply:
Reverted it to how it was, not sure why I removed the extension hook, looks like an oversight by me